docs/backup_workstations.rst

```diff
@@ -118,16 +118,92 @@ Then, copy the contents of the device's persistent volume to the directory using
-USB drive, unmount the Backup partition and store the drive somewhere safely.
+USB drive, unmount the Backup partition by clicking its Eject button. Wait until
+the Backup USB can be safely removed, and store it somewhere safely.
+
+.. note:: After the Eject button is clicked, it may be take some time before 
+          the drive can be safely removed. Wait until its entry  is removed from
+          the lefthand column of the file manager.
+
+.. _restore_workstations:
+
+Restoring a Workstation from a Backup
+-------------------------------------
+
+To recreate a backed-up *Admin Workstation*, *Journalist Workstation*, or 
+*Secure Viewing Station* Tails USB,  you will need 
+
+- your Backup USB containing the persistent volume to be restored,
+- a blank USB stick to be set up as the new workstation USB,  
+- an airgapped machine and a USB with Tails already installed, referred to as 
+  the host Tails USB in this document. The host Tails USB is only used to 
+  transfer files between the Backup USB and the new workstation USB.
+
+The process will require 3 USB ports - if necessary, you can use a USB hub. We
+recommend labeling USB devices before use, as it can be easy to confuse them.
+
+Prepare the new Tails USB
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the guide to :ref:`creating a Tails USB <set_up_tails>` to install
+Tails and create a persistent volume on the blank USB stick to create the new 
+workstation USB.
+
+ 
+Open the Backup USB and new Tails Persistent Volume
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, boot up the host Tails USB on the airgapped machine, making sure to set
+an administration password on the Tails Greeter dialog.
+
+Then, navigate to **Places** ▸ **Computer** to open the file manager, and insert
+the Backup USB. Click its entry in the lefthand column and enter its decryption
+passphrase when prompted. Its volume name (``Backup`` in the instructions above)
+will appear in place of the generic ``N.M GB Encrypted`` name.
+
+Next, insert the new workstation USB, and click its entry in the lefhand column. When
+prompted, enter its persistent volume's passphrase. The volume name ``TailsData``
+will appear in the lefthand column.
+
+Copy the Backup to the New Workstation USB's Persistent Volume
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Open a terminal by navigating to **Applications** ▸ **Favorites** 
+▸ **Terminal** . Next, use the ``rsync`` command to copy the appropriate backup
+folder to the new workstation USB's persistent volume. For example, if the backup
+folder to be copied is named ``admin-backup``, run the following command:
+
+.. code:: sh
+ 
+  sudo bash -c "rsync -a --info=progress2 --no-specials --no-devices \
+      /media/amnesia/Backup/admin-backup/ /media/amnesia/TailsData && sync"
+
+.. note:: Note that the first path argument ends with a ``/``, but the second 
+          path argument should not!
```

Comment:
Nit: The trailing slash on the _second_ argument makes no difference: with or without, the _contents_ of `/media/amnesia/Backup/admin-backup/` (but not the `admin-backup` dir itself) will be copied into `/media/amnesia/TailsData/`. 

The trailing slash on the _first_ argument is indeed crucial. Perhaps we can place a trailing slash on both args, for consistency, and update the "note" to something like:

> The directory names in the command above end with a ``/`` to ensure that only the contents of the directories are transferred, without recreating the parent directories at the destination. If you omit the trailing ``/``, the files will not be copied to the correct location for Tails to operate.